core/domain/suggestion_registry.py

```diff
@@ -427,12 +507,19 @@ def get_change_list_for_accepting_suggestion(self):
         old_content = (
             exploration.states[self.change.state_name].content.to_dict())
 
-        change.old_value = old_content
+        # Here, change is of type ExplorationChange and all attributes
+        # on ExplorationChange are created dynamically except cmd, so due
+        # this MyPy is unable to recognize `old_value` as an attribute
+        # of change and throwing `"ExplorationChange" has no attribute
```

Comment:
```suggestion
        # on ExplorationChange are created dynamically except cmd, so because
        # of this MyPy is unable to recognize `old_value` as an attribute
        # of change and throws an `"ExplorationChange" has no attribute
```

Reply:
Done!